cmake/faceliftMacros.cmake

```diff
@@ -75,7 +75,12 @@ function(facelift_add_unity_files TARGET_NAME VAR_NAME)
         file(WRITE ${UNITY_GENERATOR_FILE_LIST} ${UNITY_GENERATOR_INPUTS})
         get_target_property(UNITY_LOCATION unity_generator LOCATION)
         execute_process(COMMAND ${FACELIFT_PYTHON_EXECUTABLE} ${UNITY_LOCATION} --output ${CMAKE_CURRENT_BINARY_DIR} --target_name ${TARGET_NAME} --file_list ${UNITY_GENERATOR_FILE_LIST}
-                        OUTPUT_VARIABLE COMPLETE_FILE_LIST)
+                        OUTPUT_VARIABLE COMPLETE_FILE_LIST
+                        ERROR_VARIABLE UNITY_GENERATOR_ERROR
+                        RESULT_VARIABLE UNITY_GENERATOR_RETURN_CODE)
+        if(NOT "${UNITY_GENERATOR_RETURN_CODE}" STREQUAL "0")
+            message(FATAL_ERROR "Facelift unity files generation failed!\n} with PYTHONPATH=$ENV{PYTHONPATH}\nCommand:${FACELIFT_PYTHON_EXECUTABLE} ${UNITY_LOCATION} --output ${CMAKE_CURRENT_BINARY_DIR} --target_name ${TARGET_NAME} --file_list ${UNITY_GENERATOR_FILE_LIST}\nReturn code: ${UNITY_GENERATOR_RETURN_CODE}\nOutput: ${COMPLETE_FILE_LIST}\nError: ${UNITY_GENERATOR_ERROR}\n")
```

Comment:
do not duplicate the command line definition.
define the command line as a variable like it is done for the code generator.